exp/controllers/helpers_test.go

```diff
@@ -60,7 +60,7 @@ func TestAzureClusterToAzureMachinePoolsMapper(t *testing.T) {
 	mapper, err := AzureClusterToAzureMachinePoolsMapper(context.Background(), fakeClient, scheme, logr.New(sink))
 	g.Expect(err).NotTo(HaveOccurred())
 
-	requests := mapper(&infrav1.AzureCluster{
+	requests := mapper(context.TODO(), &infrav1.AzureCluster{
```

Comment:
Should we use `context.Background()` instead?

Reply:
Yes, I think you're right: `context.TODO()` is kind of like "I don't know what context to use here," but `context.Background()` is probably more appropriate in tests.

Reply:
Done.